content/news/playtest-20160403.md

```diff
@@ -0,0 +1,95 @@
+* Engineers now have a lower selection priority so that they will not be selected along with your regular combat units.
+* Fragile diplomacy has been removed.
+* By popular demand, a tech level without super-powers has been added to the Red Alert mod.
+* Lots of minor fixes, UI polish and performance improvements.
+
+As usual, both the Tiberian Dawn and Red Alert mod have received some balance changes. A complete list of these can be found [in the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/<hash>).
+
+<div style="text-align:center" markdown="1">
+![Restart dialog](/images/news/20160403-cnc-restart-dialog.png)
+<p>
+Players finally have a restart option after failing a mission or skirmish!
+</p>
+
+![Team colors and health bar improvements](/images/news/20160403-ra-team-colors.png)
+<p>
+When team colors are enabled in the settings, they will now be used on the radar map.<br/>
+You can also set health bars to only appear when units are damaged.
+</p>
+
+![Nod units with proper color](/images/news/20160403-cnc-nod-colors.png)
+<p>
+This is for the nostalgic ones among us (and who isn't?).<br/>
+Nod units in missions now show their proper silver colors known from the original game.
+</p>
+
+![Worm trails](/images/news/20160403-d2k-worm-trail.png)
+<p>
+Worms have received some polish and now leave a trail similar to the one from the original game.
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &mdash; from 7 to 11 &mdash; during this development cycle!
+
+We will cover these changes in more detail in the news post for the third playtest, so here is just a short overview in list format:
+
+* Custom maps are now stored in versioned directories.
+* The game will not upgrade maps automatically anymore. Map authors can upgrade their map on the resource site with the click of a button, or do it manually by using the OpenRA utility.
+* The settings from the Options: block have been moved to their respective traits.
+* Custom rules, weapon or sequence definitions can now be stored in external files.
+
+Support for third-party mods will significantly improve with the next release. It is now possible to create an archive of your whole mod (called ".oramod"). Installing it on other computers is then just a matter of placing that archive in the "mods" subdirectory of the Support folder.
+
+The news post for the second playtest will detail what changes you need to make to your mod to make it compatible with the upcoming release and to enable the .oramod packaging. All shipped mods have already been converted, so feel free to take a look at those.
+
+In the future, OpenRA will feature an ingame mod browser that will allow you to download and install mods from the resource site.
+
+<hr />
+## Dedicated server changes
```

Comment:
:+1: for the wording in this section.  Maybe include a screenshot of the server output running in a terminal window (showing a couple of people joining, changing map, starting and ending a game) to break up the text between this and the next section?